src/Components/Aspire.OracleManagedDataAccess/AspireOracleManagedDataAccessExtensions.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data.Common;
+using Aspire;
+using Aspire.OracleManagedDataAccess;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Oracle.ManagedDataAccess.Client;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Extension methods for connecting Oracle database with Oracle.ManagedDataAccess client
+/// </summary>
+public static class AspireOracleManagedDataAccessExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:OracleManagedDataAccess";
```

Comment:
```suggestion
    private const string DefaultConfigSectionName = "Aspire:Oracle:ManagedDataAccess:Core";
```

This should follow the name of the component.

Reply:
Sure